### ISSUE TITLE
[BUG] Calendar month detection

### ISSUE THREAD

**Issue by VasKho** [elkowar/eww#625]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

I face the problem with `calendar` widget. I'm trying to make `:onclick` event and found that `{2}` variable returns not my current month (i.e. '10' instead of '11').

### Reproducing the issue

Here is my minimal problematic config

```lisp
(defpoll month-year :interval "24h" "date +'%B %Y'")

(defwindow calendar
  :monitor 0
  :geometry (geometry :x "0"
                      :y "30px"
                      :width "13%"
                      :height "30px"
                      :anchor "top right")
  :stacking "fg"
  :exclusive false
   (cal))

(defwidget cal []
  (box :class "cal-box" :orientation "v"
       (box :orientation "v" :class "cal-inner-box" :space-evenly false
	    (box :orientation "h" :class "monthyear" "${month-year}")
	    (box :class "day-names" :orientation "h" :space-evenly true
		 (label :text "Su")
		 (label :text "Mo")
		 (label :text "Tu")
		 (label :text "We")
		 (label :text "Th")
		 (label :text "Fr")
		 (label :text "Sa"))
	    (calendar :class "cal" :show-details false :show-heading false :show-day-names false
                      ;; I have to fix this with $(({1}+1)) instead of {1}
		      :onclick "${eww} update month-year=\"$(date -d {2}-{1}-{0} +'%B %Y')\""))))

```
Specifying `:day`, `:month` and `:year` properties doesn't help as well.

### Expected behaviour

Correct month detection

### Additional context
* There is another closed issue #544 that might be connected with current problem
* I use AUR version [eww compiled with wayland support](https://aur.archlinux.org/packages/eww-wayland-git)

_No response_

**Comment by Swexti** [comment]
This was fixed in the latest 0.4.0 version of Eww released in september. You'll have to build it manually to update, the last update of that AUR-package was in August.

**Comment by VasKho** [comment]
Oh, thanks for your help
